src/index.js

```diff
@@ -2,25 +2,37 @@ import ReactDOM from 'react-dom'
 import { BrowserRouter, Routes, Route } from 'react-router-dom'
 import Layout from './pages/Layout'
 import Home from './pages/Home'
-import About from './pages/About'
-import Contact from './pages/Contact'
-import Services from './pages/Services'
+// import About from './pages/About'
+// import Contact from './pages/Contact'
+// import Services from './pages/Services'
```

Comment:
Question(nb, I'm not sure I understand what you're trying to do): Is there a reason you're commenting out the imports instead of deleting them? I'd prefer we delete things if we think we are not going to use them soon - we can always import them again!

Reply:
I didn’t delete them because there is another card on Jira for removing them. But will delete it when I’m fixing the other part of the code you commented 😊